README.md

```diff
@@ -829,6 +829,29 @@ bundle | string | yes | Bundle ID of the XCTest | 'com.bundle.myapp'
 
 Array of xctests in the test bundle (e.g.: `[ 'XCTesterAppUITests.XCTesterAppUITests/testExample', 'XCTesterAppUITests.XCTesterAppUITests/testLaunchPerformance' ]`)
 
+### mobile: viewportRect
+
+Return the rectangle of viewport.
+The viewport is the device screen size without the status bar.
```

Comment:
The viewport is the device's screen size with status bar size subtracted if the latter is present/visible